src/backend/.config/flag.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
How about renaming the file to `config.json`?

Reply:
Infact,  `config.local.json` since it only corresponds to local development.

Reply:
great idea - no confusion